docs/apm-input-settings.asciidoc

```diff
@@ -189,7 +189,7 @@ If true, APM Server captures the IP of the instrumented service and its User Age
 // tag::rum_allow_origins-setting[]
 |
 [id="input-{input-type}-rum_allow_origins-setting"]
-`rum_allow_origins`
+Origin Headers
```

Comment:
Maybe `Allowed Origins`? I'm not sure. Maybe it's obvious to people who work with CORS daily, but saying "Origin Headers" doesn't really explain to me that it's an allow list.

Reply:
```suggestion
Allowed Origins
```